docs/source/contents/running_with_docker.md

```diff
@@ -0,0 +1,33 @@
+# Running with Docker
+
+## Use the pre-built image
+You can easily run NWB widgets Panel using Docker.
+
+```bash
+$ docker pull ghcr.io/NeurodataWithoutBorders/nwbwidgets-panel:latest
+$ docker run -p 8866:8866 nwbwidgets-panel
```

Comment:
I'd love to test this - can you either get this to trigger from this branch - or manually push your build [[instructions](https://github.com/catalystneuro/neuroconv/pull/383/files#diff-fcc22aa372714b7faa21a54039c3a7fbe1abd106c1f15f9f8e6cf85588dfe592R28-R47)]